testcafe/tests/helpers/visualreview-utils.js

```diff
@@ -0,0 +1,46 @@
+import VisualReview from 'visualreview-client'
+import { t } from 'testcafe'
+import { getNavigatorOs, getNavigatorName, getResolution } from './utils'
+
+//Put this const in travis after POC
+const VISUALREVIEW_INSTANCE = 'visualreview.cozycloud.cc'
+
+export class VisualReviewTestcafe extends VisualReview {
+  constructor(options) {
+    super(options)
+    this.options.protocol = 'https'
+    this.options.hostname = VISUALREVIEW_INSTANCE
+  }
+
+  async takeScreenshotAndReview(imageName) {
+    await t.takeScreenshot(imageName)
+
+    this.options.properties.os = await getNavigatorOs()
+    this.options.properties.browser = await getNavigatorName()
+    this.options.properties.resolution = await getResolution()
+
+    //the path needs to be in const but i need to define the screenshots tree 1st
+    this.uploadScreenshot('./reports/screenshots/' + imageName)
+  }
+
+  async checkVr() {
```

Comment:
Maybe we can find a better name, like `checkStatus`? We're in the VisualReview class so we know it's something about `vr`.